spatial-lib/src/cmr/spatial/lr_binary_search.clj

```diff
@@ -267,3 +267,15 @@
            (warn "Use mbr from one of the points in the polygon because lr is not found "
                  (pr-str polygon))
            (m/point->mbr (-> polygon :rings first :points first))))))))
+
+(comment
```

Comment:
Not sure if you want this comment block here still

Reply:
It is very useful for debugging. I intentionally kept it.